NETProvider/src/FirebirdSql.Data.UnitTests/Properties/AssemblyInfo.cs

```diff
@@ -22,7 +22,7 @@
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
-[assembly: CLSCompliant(true)]
+[assembly: CLSCompliant(false)]
```

Comment:
The compiler adds warning about ```[TestFixture(FbServerType.Embedded)```
See
https://msdn.microsoft.com/en-us/library/h9f682z5%28VS.80%29.aspx?f=255&MSPPError=-2147217396
This is because TestFixture attribute takes parameters as an array. (https://github.com/nunit/nunit/blob/master/src/NUnitFramework/framework/Attributes/TestFixtureAttribute.cs#L53)